functions.gallery.php

```diff
@@ -7,6 +7,15 @@ class Jetpack_Gallery_Settings {
 	function __construct() {
 		add_action( 'admin_init', array( $this, 'admin_init' ) );
 	}
+	
+	/**
+	 * Renders more descriptive name for the default gallery type.
+	 *
+	 * @Since 2.5.1
+	 *
+	 * @param array  $instance An array of gallery types.
```

Comment:
It seems the array actually only contains the default gallery type, "Thumbnail Grid". Might be worth mentioning.